Examples/Python/tests/helpers/__init__.py

```diff
@@ -74,6 +77,11 @@
 
 isCI = os.environ.get("CI", "false") == "true"
 
+if isCI:
+    for k, v in dict(locals()).items():
+        if k.endswith("Enabled"):
+            locals()[k] = True
```

Comment:
wow that looks evil :smile:

Reply:
That way it turns on everything `*Enabled`, and we can't forget by accident.

Reply:
I guess one down side is that we will not know if the flags are corrupted again. do you think it makes sense to assert them to be turned instead of silently turning them on?

Reply:
That might be another alternative, yeah.